test/multiverse/suites/config_file_loading/config_file_loading_test.rb

```diff
@@ -199,6 +199,34 @@ def test_config_will_load_settings_for_environment_passed_manual_start
     assert_equal 'bazbangbarn', NewRelic::Agent.config[:i_am], "Agent.config did not load bazbangbarn config as requested"
   end
 
+  def test_parses_default_settings_correctly
+    config_contents = <<-YAML
+common: &default_settings
+  license_key: 47ae68e8919929c1d99fa3a1a36a0aa8c06ba690
```

Comment:
Since this is already committed, we may need to do some finagling, but I wonder if we can pull this license key out?